devito/core/arm.py

```diff
@@ -21,7 +21,7 @@ def _specialize_iet(cls, graph, **kwargs):
             mpiize(graph, mode=options['mpi'])
 
         # Lower IncrDimensions so that blocks of arbitrary shape may be used
-        relax_incr_dimensions(graph, sregistry=sregistry)
+        finalize_loop_bounds(graph)
```

Comment:
I don't like the new name. Why did it change?

Reply:
I think you told me that we are not "relaxing" anymore so we should think of something different